source/sass/blocks/reviews.scss

```
@@ -56,3 +81,32 @@
     display: none;
   }
 }
+
+.reviews__arrow {
+  @media (max-width: $mobile-width-only) {
+    display: none;
+  }
+
+  @media (min-width: $tablet-width) {
+    display: none;
+  }
+
+  @media (min-width: $desktop-width) {
+    display: block;
```

Как-то непонятно, зачем сперва отключать только на телефоне, а потом на планшете и выше, и на десктопе включить. Можно же проще:
```suggestion
.reviews__arrow {
  display: none;

  @media (min-width: $desktop-width) {
    display: block;
```